tests/transaction_test.go

```diff
@@ -3,10 +3,9 @@ package tests_test
 import (
 	"context"
 	"errors"
-	"testing"
-
 	"gorm.io/gorm"
 	. "gorm.io/gorm/utils/tests"
+	"testing"
```

Comment:
Hello @a631807682 

Seems like this code is not formatted with `gofmt`?

Reply:
@jinzhu 
It's formatted by my gofmt, it is formatted like this after I run `gofmt -s -w ./tests`.
I will try to understand why it happens.
```shell
> go version
go version go1.18 darwin/arm64
```

```shell
> gofmt -e -d ./tests
diff -u tests/transaction_test.go.orig tests/transaction_test.go
--- tests/transaction_test.go.orig      2022-05-17 12:50:51.000000000 +0800
+++ tests/transaction_test.go   2022-05-17 12:50:51.000000000 +0800
@@ -3,9 +3,9 @@
 import (
        "context"
        "errors"
-       "testing"
        "gorm.io/gorm"
        . "gorm.io/gorm/utils/tests"
+       "testing"
 )
 
 func TestTransaction(t *testing.T) {
```

Reply:
refer to https://github.com/mvdan/gofumpt/issues/228